hpcm/hpe-mlds-build-hpcm-os-image.md

```diff
@@ -195,18 +207,24 @@ master_host: $(/opt/clmgr/bin/cm node show -n master -M --display-no-header | aw
 master_port: 8080
 resource_pool: aux-pool
 EOF
+chmod 640 /etc/determined/agent.yaml
```

Comment:
To protect any sensitive information that may end up in /etc/determined/agent.yaml now, or in the future.